sdk/core/azure_core_amqp/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corp. All Rights Reserved.
```

Comment:
Nit: full header should go here. See files under `sdk/typespec` in the `feature/track2` branch now. Eventually we'll enforce this with linting like some other languages do. Be sure to install the `EditorConfig` extension in VSCode. We do have a `.editorconfig` file that should help with this (not an error currently, though).

Reply:
See <https://docs.opensource.microsoft.com/releasing/general-guidance/copyright-headers/>.

/cc @RickWinter @ronniegeraghty as an FYI.